bedrock/base/templates/base-resp.html

```diff
@@ -7,6 +7,9 @@
   <head>
     <meta charset="utf-8">{# Note: Must be within first 512 bytes of page #}
 
+    {% javascript 'site' %}
+    {% block newrelic %}{% endblock %}
```

Comment:
I can't think of any reason moving the JS block here would cause any problems. Should be fine. Could even potentially improve rendering, as platform will be known prior to parsing CSS (for platform-specific rules and such).